### PR TITLE
git sync for AWS::Events::EventBus

### PR DESCRIPTION
**Pull request by aungcemi**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by hlazrv** [comment]
Can you run pre-commit run --all-files

File: aws-events-eventbus/contract-tests-artifacts/testAccountsConfig.yml
Comment (hlazrv) [inline aws-events-eventbus/contract-tests-artifacts/testAccountsConfig.yml:1]: Remove this file

File: aws-events-eventbus/scripts/generate-test-accounts-config.sh
Comment (hlazrv) [inline aws-events-eventbus/scripts/generate-test-accounts-config.sh:1]: Remove this file

Comment (hlazrv) [inline aws-events-eventbus/scripts/generate-test-accounts-config.sh:1]: Remove this file

File: aws-events-eventbus/settings.internal.json
Comment (hlazrv) [inline aws-events-eventbus/settings.internal.json:1]: Remove this file

**Review by hlazrv — APPROVED** [review]

**Review by ceskander-aws — APPROVED** [review]

**Pull request merged**